media/chat.css

```diff
@@ -2,7 +2,7 @@
 .sidebar__section-container {
   transition: all 0.3s ease;
   overflow: auto;
-  max-height: 70vh;
+  max-height: 85vh;
```

Comment:
**suggestion:** the box with the recipes is a little big by default, I don't know if it's here that should be changed though

Reply:
Ah great spot - removed the left and right margins and I think this is resolved.